runtime/test/planner-tests.js

```diff
@@ -170,7 +173,7 @@ describe('ConvertConstraintsToConnections', async () => {
     let {result, score} = results[0];
     assert.deepEqual(result.toString(),
 `recipe
-  create as view0
+  create as view0 // S
```

Comment:
What's the `// S`?

Reply:
(more instances below).

Reply:
`// S` is the type (like in real recipes that would be, for example: // Product) - see `schema S` above.
I'm debating whether i should rename it to `schema Thing` for example, but then it won't be consistent with the one letter particle names, like `particle A`.

Reply:
Ah, I get it. I think (although I don't quite remember) I thought this was supposed to be `#S`, so more detail in the comment might help.

Reply:
It was # S when we used # (and not //) for comments.